src/main/java/seedu/duke/commands/Command.java

```diff
@@ -1,14 +1,17 @@
 package seedu.duke.commands;
 
 import seedu.duke.exceptions.InstantiateAbstractClassException;
+import seedu.duke.exceptions.ModHappyException;
 
 /**
  * Parent class of all commands in Mod Happy.
  */
 public abstract class Command {
     protected String commandName = "Command";
 
-    public String execute() throws InstantiateAbstractClassException {
```

Comment:
Since `InstantiateAbstractClassException` is not used anymore (and anyway Java has its own built in exception for this) maybe we can just delete the class